internal/util/fsql/tx.go

```diff
@@ -116,8 +127,10 @@ func (tx *Tx) ExecContext(ctx context.Context, query string, args ...any) (sql.R
 		ra = &rav
 	}
 
-	fields = append(fields, zap.Int64p("rows", ra), zap.Duration("time", time.Since(start)), zap.Error(err))
-	tx.l.Sugar().With(fields...).Debugf("<<< %s", query)
+	fields = append(fields, slog.Any("rows", ra), slog.Duration("time", time.Since(start)), logging.Error(err))
```

Comment:
`slog.Any("rows", ra)` is unlikely to be handled correctly by any handler

Reply:
True, I've only tested in `console` handler, but you are right about other handlers unable to handle it.

Reply:
I've changed the logic a bit, `rows` attr is added when exec is successful, only when there is no error. There is no need to set the attribute if its value is nil in my opinion.